### PR TITLE
Itea documentation

### PR DESCRIPTION
**Pull request by UkrainianEngineer**
Added some basic information.

### CODE REVIEW

**Comment by RomanPryima** [comment]
Thanks for this guide.
We've been needed it from the start of the work.

**Comment by kate-ka** [comment]
Thanks, useful guide .

File: docs/procedures/working_with_branches.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Last 2 commands merges changes from `release` into `<name_of_feature_branch>`.
+
+Merging conflicts
+~~~~~~~~~~~~~~~~~
+
+If the same parts of files changed in two separate branches, merging conflicts
+will appear during merging these branches.
+
+Conclicts resolution
```
Comment (Sasha1152) [inline docs/procedures/working_with_branches.rst]: Con-f-licts

Comment (UkrainianEngineer) [reply docs/procedures/working_with_branches.rst]: Thanks! Fixed!

**Comment by UkrainianEngineer** [comment]
@RomanPryima , @kate-ka , @Sasha1152 

Guys, could you please set 👍 or add some comment into this review? Thanks!

**Comment by Sasha1152** [comment]
:+1:

**Comment by kate-ka** [comment]
:thumbsup:

**Pull request merged**